posthog/session_recordings/queries/test/session_replay_sql.py

```diff
@@ -94,7 +94,7 @@ def _sensible_last_timestamp(
             if isinstance(first_timestamp, str):
                 first_timestamp = parse(first_timestamp)
 
-            sensible_timestamp = (first_timestamp - relativedelta(seconds=3600)).isoformat()
+            sensible_timestamp = (first_timestamp + relativedelta(seconds=3600)).isoformat()
```

Comment:
past me as over confident calling something `sensible_timestamp` since they then set it to something silly because of copy pasta 🙈